modules/ting_field_search/ting_field_search.module

```diff
@@ -611,6 +606,11 @@ function ting_field_search_form_search_block_form_alter(&$form, &$form_state) {
   }
 
   $profile = ting_field_search_get_default_profile();
+
+  if (strpos(current_path(), 'search/node') !== FALSE) {
```

Comment:
I think more correct would be `strpos(current_path(), 'search/node') === 0` so you're actually verifying that current_path starts with 'search/node'?

Reply:
\+